File: Java/Primos.java
import java.util.Scanner;

public class Primos {

  public void chamarPrimos(){

    int aux = 0;
    Scanner prompt = new Scanner(System.in);

    System.out.println("Digite um número: ");
    int num = prompt.nextInt();

    for (int i = 2; i <= num / 2; i++) {
      if (num % i == 0) {
        aux++;
      }
    }

    if (aux >= 1) {
      System.out.println("O número " + num + " não é primo");
      } else {
      System.out.println("O número " + num + " é primo");
      }
    }
}

File: primos.js
var num = prompt("Digite um numero para conferencia se é primo: ")
var aux = 0;

for (var i = 2; i <= num / 2; i++) {
  if (num % i == 0) {
    aux += 1;
  }
}
/*
if(aux >= 1){
    console.log('O numero não é primo')
}
else {
    console.log("O numero é primo")
}
*/

if (aux >= 1) {
  document.write('O numero não é primo' + "<br>")
}
else {
  document.write("O numero é primo" + "<br>")
}
